lib/core/presentation/pages/notification/widgets/notification_card_widget.dart

```diff
@@ -76,7 +76,7 @@ class NotificationCard extends StatelessWidget {
               color: colorScheme.onPrimary,
               fontWeight: FontWeight.w700,
             )),
-        Text(timeago.format(notification.stamp!, locale: 'en_short'),
+        Text(timeago.format(notification.createdAt!, locale: 'en_short'),
             style: Typo.medium.copyWith(
```

Comment:
In the future for some case like this, you might consider changing their jsonKey instead of refactor name. It will reduce code changes on entire the project

Reply:
I see, good point 👍 Thanks